src/main/java/org/openhmis/manager/TokenCacheManager.java

```diff
@@ -0,0 +1,112 @@
+package org.openhmis.manager;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+
+import org.openhmis.dao.TmpTokenCacheDAO;
+import org.openhmis.domain.TmpTokenCache;
+import org.openhmis.dto.TokenCacheDTO;
+
+public class TokenCacheManager {
```

Comment:
Nobody reading this code will know what the TokenCache is all about, unless the person is you or has had a conversation with you.  We need to document what this thing is for, how one uses it, and what promises it makes or doesn't make.